packages/fluentui/react/src/themes/teams/components/Toolbar/toolbarRadioGroupStyles.ts

```diff
@@ -4,6 +4,7 @@ import { ToolbarVariables } from './toolbarVariables';
 
 const toolbarRadioGroupStyles: ComponentSlotStylesPrepared<ToolbarRadioGroupStylesProps, ToolbarVariables> = {
   root: (): ICSSInJSStyle => ({
+    display: 'flex',
```

Comment:
It's a required otherwise `ToolbarDivider` will be broken inside `ToolbarRadioGroup`